src/main/java/frc/robot/subsystems/intake/IntakeRollerCommandsBuilder.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.subsystems.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+
+import java.util.function.DoubleSupplier;
+
+public class IntakeRollerCommandsBuilder {
+
+	private IntakeRoller intakeRoller;
+
+	public IntakeRollerCommandsBuilder(IntakeRoller intake) {
+		this.intakeRoller = intake;
+	}
+
+	public Command setPower(double power) {
```

Comment:
add some spaces
hint: put //@formatter:off here in formatter:on after all of the commands

Reply:
itd funny here, look at others code and copy paste